docs_italia_theme/__init__.py

```diff
@@ -172,9 +177,9 @@ def generate_additonal_tocs(app, pagename, templatename, context, doctree):
             figure_number = toc_fig_tables.get(figure_id)
             if figure_number is None:
                 continue
-            figure_title = figurenode.children[0].get('alt') or context['t']['no_description']
+            figure_title = figurenode.children[-1].children[0] or context['t']['no_description']
```

Comment:
Curiosità: questa è stata cambiata così perché hai bisogno di identificare elementi in modo diverso nel DOM?

Reply:
Hai ragione, questa andava spiegata meglio.
Anziché usare il testo alternativo delle immagini come testo per la toc, usiamo la didascalia.
Questo perché il testo alternativo potrebbe essere molto lungo perché contiene una descrizione estesa della figura (così è ad esempio nel nuovo PT).
Si tratta di una breaking change, ma non avrà impatto perché l'unico documento ad utilizzare attualmente questa feature è la guida di Docs che aggiornerò in giornata per documentare proprio questo cambiamento.